lib/commands.js

```diff
@@ -2,8 +2,15 @@
 
 'use strict';
 
-var buildPolicyString = require('./utils')['buildPolicyString'];
-var chalk = require('chalk');
+const chalk = require('chalk');
+const {
+  buildPolicyString,
+  calculateConfig,
+  readConfig
+} = require('./utils');
+
+const CSP_HEADER              = 'Content-Security-Policy';
```

Comment:
Definitely non-blocking, and probably just a bit too nitpicky but _in general_ I prefer to avoid the equals alignment stuff. It makes future changes change lines that are unrelated (e.g. when adding new variables with a slightly longer name), and then makes it just a tiny bit harder to follow whats going on through the git history.

Reply:
:100: I was just adopting the existing coding style.